lib/pbench/test/unit/server/conftest.py

```diff
@@ -129,7 +129,20 @@ def server_config(on_disk_server_config, monkeypatch) -> PbenchServerConfig:
 
 
 @pytest.fixture()
-def client(monkeypatch, server_config, fake_email_validator):
+def set_oidc_well_known_endpoints(monkeypatch):
+    def fake_well_known(oidc_client):
+
+        oidc_client.USERINFO_ENDPOINT = "https://oidc_userinfo_endpoint.example.com"
+        oidc_client.TOKENINFO_ENDPOINT = "https://oidc_token_introspection.example.com"
+        oidc_client.JWKS_URI = "https://oidc_jwks_endpoint.example.com"
+
+    monkeypatch.setattr(OpenIDClient, "set_well_known_endpoints", fake_well_known)
+
+
+@pytest.fixture()
+def client(
+    monkeypatch, server_config, fake_email_validator, set_oidc_well_known_endpoints
+):
```

Comment:
I don't think we need `monkeypatch` here, anymore (i.e., you removed the reference to it at old line 153 and replaced it with `set_oidc_well_known_endpoints` above).